React/Base/RCTBridge.m

```diff
@@ -823,9 +823,15 @@ - (void)reload
   /**
    * AnyThread
    */
+  typeof(self) __weak weakSelf = self;
   dispatch_async(dispatch_get_main_queue(), ^{
-    [self invalidate];
-    [self setUp];
+    typeof(self) strongSelf = weakSelf;
+    if (!strongSelf || !strongSelf.isValid) {
+      return;
+    }
+
+    [strongSelf invalidate];
+    [strongSelf setUp];
```

Comment:
You don't need `strongSelf` here, it can be only:

```objc
__weak RCTBridge* weakSelf = self;
dispatch_async(dispatch_get_main_queue(), ^{
  if (!weakSelf.isValid) {
    return;
  }

  [weakSelf invalidate];
  [weakSelf setUp];
});
```